src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.h

```diff
@@ -77,6 +78,7 @@ class ThrusterDynamicEffector: public SysModel, public DynamicEffector {
     Eigen::MatrixXd* inertialPositionProperty;  //!< [m] r_N inertial position relative to system spice zeroBase/refBase
     StateData *fuelMass;                           //!< pointer to the fuel tank mass state
     std::string nameOfMassState{};                 //!< name of mass state
+    double maxFuelMass;                            //!< maximum fuel capacity used for tank percentage
```

Comment:
This variable is used in "Add the fuel mass state to thrusterDynamicEffector" commit

Reply:
Ah yes, thanks for catching this. I've moved the fuel tank's use of the variable out of the "Add the fuel mass to thrusterDynamicEffector" commit and into the "Implement blow down in thrusterDynamicEffector" commit where it is created here on the thruster side.